<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="readparameterpB">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>读取参数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectpB">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="parametersetpB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendaxispB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送坐标</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>坐标值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>目标数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>每圈距离（mm）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>吹气持续时间（mm）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="aimnumLE">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="roubddistanceLE">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="timedurationLE">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendmodeLE">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>是否是发送模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>皮带速度（mm）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="speedLE">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>ip地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipadressLE">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="axisLE">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>601</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="messagedisTB">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>69</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>报文显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moderead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>模式读取</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
